SU-SWS/saa_victoria_subtheme
# Adapt 286  buttons & cta styles

# READY FOR REVIEW

# Summary
- Adjust link colors, and icon colors. Removed text-decoration and added hover :pointer for links.

# Review By (Date)
- This week asap, June 5th please

# Urgency
- low

# Steps to Test

1. pull down branch
1. create buttons and links with this code pasted in a wywiwyg text paragraph:
```
<p class="plain-text"><a class="su-button" href="&lt;front&gt;">Button</a></p>

<p class="plain-text"><a class="su-button su-button--secondary" href="&lt;front&gt;">Secondary Button</a></p>

<p class="plain-text"><a class="su-button su-button--big" href="&lt;front&gt;">Big Button</a></p>

<p class="plain-text">Here is another paragraph afterwards.</p>
<p class="plain-text"><a class="su-link" href="&lt;front&gt;">CTA Link</a></p>

<p class="plain-text"><a class="su-link su-link--action" href="&lt;front&gt;">CTA Link Action</a></p>

<p class="plain-text"><a class="su-link su-link--external" href="&lt;front&gt;">CTA Link External</a></p>

<p class="plain-text"><a class="su-link su-link--download" href="&lt;front&gt;">CTA Link Download</a></p>

<p class="plain-text"><a class="su-link su-link--video" href="&lt;front&gt;">CTA Link Video</a></p>

<p class="plain-text"><a class="su-link su-link--jump" href="&lt;front&gt;">CTA Link Jump</a></p>

<p class="plain-text"><a class="su-link su-link--top" href="&lt;front&gt;">CTA Link Back to Top/Collapse</a></p>

<p class="plain-text"><a class="su-link su-link--internal" href="&lt;front&gt;">CTA Link Internal</a></p>
```
2. Then see if styles match figma: https://www.figma.com/file/TXNBECAGcEkZ3vwc6xcZ3g/SAA-Design-System?node-id=192%3A384

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- ADAPT-286

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
